R/execution.r

```diff
@@ -78,11 +78,17 @@ Executor <- setRefClass(
 is.silent = function() {
     current_request$content$silent
 },
+is.store_history = function() {
+    current_request$content$store_history
+},
+request_metadata.get = function() {
```

Comment:
I see what you are trying to do here, but I really dislike the mixing of `_` and `.`.

`should.store.history` and `get.request.metadata` ?